tests/unit/services/intl-test.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable ember/new-module-imports */
```

Comment:
I *think* you could use https://github.com/rwjblue/ember-debug-handlers-polyfill instead of the `Ember.warn` override below, but seem fine to explore that in a separate PR